air/src/proof.rs

```diff
@@ -94,6 +95,8 @@ pub enum HashFunction {
     Blake3_256 = 0x01,
     /// RPO hash function with 256-bit output.
     Rpo256 = 0x02,
+    /// RPX hash function with 256-bit output.
+    Rpx256 = 0x03,
```

Comment:
We should also update the `TryFrom` implementation on line 128 below. Otherwise, the verifier panics when we try to verify proofs generated with RPX.